frontend/lib/norent/letter-content.tsx

```diff
@@ -46,6 +46,14 @@ const FullName: React.FC<NorentLetterContentProps> = (props) => (
   </>
 );
 
+export const getStreetWithApt = ({
+  street,
+  aptNumber,
+}: Pick<NorentLetterContentProps, "street" | "aptNumber">) => {
+  if (!aptNumber) return street;
+  return `${street} #${aptNumber}`;
```

Comment:
Oooh, also I'm wondering if we should remove the `#` here— there are many apartment "numbers" that aren't actually numbers (like `K` or `BSMT`) and I wouldn't want the hash symbol to affect readability on those

Reply:
Hmm so the `#` is something I've noticed that Lob does, so it seemed like a standard thing--possibly to make it easier to disambiguate from the street name on a mailing address, e.g. "123 Main Street K" might read as "house number 123 on the street called Main Street K", rather than "Apartment K at 123 Main Street".  But since this is on the letter copy rather than the mailing address, that potential ambiguity is reduced... but I'm still a bit concerned it might read weird if we don't put _some_ kind of indicator that what follows is the apt/unit/lot number-thingy, rather than part of the street name.

Hmm so I'm not sure.  I also think we should be consistent, and this JS code actually mirrors some Python code in the backend, so if we change it here we should probably change it there too.  Oy!  I'm going to stash this in #1458 for now.